front-end/src/app/shared/resolvers/report.resolver.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('ReportResolver', () => {
     httpTestingController.verify();
   });
 
-  it('should return undefined', () => {
+  xit('should return undefined', () => {
```

Comment:
This unit test is broken.  The expect() call never registers with Karma, and attempts to make it asynchronous did not work